Manager/SubscriberManager.php

```diff
@@ -0,0 +1,197 @@
+<?php
+
+/*
+ * This file is part of the MindbazBundle package.
+ *
+ * (c) David DELEVOYE <david.delevoye@adeo.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace MindbazBundle\Manager;
+
+use mbzOneshot\OneshotWebService;
+use mbzOneshot\Send;
+use mbzSubscriber\ArrayOfInt;
+use mbzSubscriber\ArrayOfString;
+use mbzSubscriber\GetSubscribersByEmail;
+use mbzSubscriber\InsertSubscriber;
+use mbzSubscriber\Subscriber as MindbazSubscriber;
+use mbzSubscriber\SubscriberWebService;
+use mbzSubscriber\Unsubscribe;
+use MindbazBundle\Exception\SendErrorException;
+use MindbazBundle\Model\Subscriber;
+use MindbazBundle\Serializer\SubscriberEncoder;
+use Psr\Log\LoggerInterface;
+use Psr\Log\NullLogger;
+use Symfony\Component\Serializer\Normalizer\DenormalizerInterface;
+use Symfony\Component\Serializer\SerializerInterface;
+
+/**
+ * @author Vincent Chalamon <vincent@les-tilleuls.coop>
+ */
+class SubscriberManager
+{
+    const MINDBAZ_SEND_RESPONSE_OK = 'OK';
+    const MINDBAZ_SEND_RESPONSE_NOK = 'NOK';
+
+    /**
+     * @var SubscriberWebService
+     */
+    private $subscriberWebService;
+
+    /**
+     * @var OneshotWebService
+     */
+    private $oneshotWebService;
+
+    /**
+     * @var SerializerInterface|DenormalizerInterface
+     */
+    private $serializer;
+
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
+
+    /**
+     * @param SubscriberWebService                       $subscriberWebService
+     * @param OneshotWebService                          $oneshotWebService
+     * @param SerializerInterface|DenormalizerInterface  $serializer
+     * @param LoggerInterface|null                       $logger
+     */
+    public function __construct(SubscriberWebService $subscriberWebService, OneshotWebService $oneshotWebService, SerializerInterface $serializer, LoggerInterface $logger = null)
```

Comment:
When the app using JMSSerializer as default Serializer, we've got this error :
Catchable Fatal Error: Argument 3 passed to MindbazBundle\Manager\SubscriberManager::__construct() must be an instance of Symfony\Component\Serializer\SerializerInterface, instance of JMS\Serializer\Serializer given